### PR TITLE
Update GitHub issue templates

### PR DESCRIPTION
**Pull request by metalmatze**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+**What happened?**
+
+**Did you expect to see some different?**
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Environment**
+
+* Prometheus Operator version:
```
Comment (squat) [inline .github/ISSUE_TEMPLATE/bug.md]: This should probably no longer be prometheus operator version but eventually `kube-prometheus` version, right?

File: .github/ISSUE_TEMPLATE/bug.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+    `Insert image tag or Git SHA here`
+    <!-- Try kubectl -n monitoring describe deployment prometheus-operator -->
```
Comment (squat) [inline .github/ISSUE_TEMPLATE/bug.md]: This will likely have to change as well

File: .github/ISSUE_TEMPLATE/support.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+**What did you do?**
+
+**Did you expect to see some different?**
+
+**Environment**
+
+* Prometheus Operator version:
+
+    `Insert image tag or Git SHA here`
+    <!-- Try kubectl -n monitoring describe deployment prometheus-operator -->
```
Comment (squat) [inline .github/ISSUE_TEMPLATE/support.md]: Ditto

**Comment by metalmatze** [comment]
I left Prometheus Operator in on purpose, due to being a dependency of kube-prometheus. Happy to discuss anyway.

**Pull request merged**